source/extensionPoints.py

```diff
@@ -142,14 +179,17 @@ class Action(HandlerRegistrar):
 
 	>>> somethingHappened = extensionPoints.Action()
 
-	Interested parties then register to be notified about this action:
+	Interested parties then register to be notified about this action, see
+	c@{HandleRegistrar.register} docstring for details of the type of handlers that can be

```

Comment:
Note that in Epydoc syntax, you probably want to use `L{some.method}`, which will generate a link.